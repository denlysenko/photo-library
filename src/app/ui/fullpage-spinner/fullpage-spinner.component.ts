import { Component, ChangeDetectionStrategy } from '@angular/core';

@Component({
  selector: 'app-fullpage-spinner',
  templateUrl: './fullpage-spinner.component.html',
  styleUrls: ['./fullpage-spinner.component.scss'],
  changeDetection: ChangeDetectionStrategy.OnPush,
})
export class FullpageSpinnerComponent {}
