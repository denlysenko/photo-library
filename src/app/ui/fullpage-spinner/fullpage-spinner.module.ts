import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';

import { FullpageSpinnerComponent } from './fullpage-spinner.component';

@NgModule({
  imports: [CommonModule],
  declarations: [FullpageSpinnerComponent],
  exports: [FullpageSpinnerComponent],
})
export class FullpageSpinnerModule {}
