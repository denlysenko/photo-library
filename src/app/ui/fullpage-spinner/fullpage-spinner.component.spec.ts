import { CommonModule } from '@angular/common';
import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { FullpageSpinnerComponent } from './fullpage-spinner.component';

describe('FullpageSpinnerComponent', () => {
  let component: FullpageSpinnerComponent;
  let fixture: ComponentFixture<FullpageSpinnerComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [CommonModule],
      declarations: [FullpageSpinnerComponent],
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(FullpageSpinnerComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(fixture).toMatchSnapshot();
  });
});
