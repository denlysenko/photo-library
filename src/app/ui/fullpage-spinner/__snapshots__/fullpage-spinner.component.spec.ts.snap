// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullpageSpinnerComponent should create 1`] = `
<app-fullpage-spinner>
  <div
    class="spinner"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</app-fullpage-spinner>
`;
